PeopleAndAppleStockPrices/PeopleAndAppleStockPrices/StockModel.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  StockModel.swift
+//  PeopleAndAppleStockPrices
+//
+//  Created by Sam Roman on 9/4/19.
+//  Copyright © 2019 Pursuit. All rights reserved.
+//
+
+import UIKit
+
+struct Stock: Codable {
+    var date: String
+    var open: Double
+    var close: Double
+    var label: String
+    var high: Double
+    var low: Double
+    
+    var sectionName: String {
+        let components = date.components(separatedBy: "-")
+        let year = components[0]
+        let month = components[1]
+        return "\(monthDictionary[month]!) \(year)"
+    }
+    
+    private let monthDictionary = ["01":"January", "02": "February", "03": "March",
```

Comment:
Very good job implementing a dictionary here! It is a great way of accessing the different months via the string number. Also I am glad you did it in the model.